packages/flutter_test/test/widget_tester_test.dart

```diff
@@ -751,6 +751,24 @@ void main() {
       expect(desktop.values.union(mobile.values), equals(all.values));
     });
   });
+
+  group('Pending timer', () {
+    test('Throws assertion message without code', () async {
```

Comment:
I just realized that it doesn't have the setUp/tearDown code?  `reportTestException` is a global, so changing it within the test would cause other tests to fail if run in a different order.

Reply:
Oh, sorry that didn't occur to me. I added setUp and tearDown again.